assessments/guide/security-reviewer.md

```diff
@@ -1,9 +1,16 @@
 # Security reviewer
 
 ## Lead reviewer
-The lead reviewer serves as the primary security assessment facilitator, often coordinating responses between the project and reviewers, ensuring communications are performed in a timely fashion as described in the [guide](./).  The lead reviewer also responsible for coordinating and recruiting additional reviewers as appropriate, reviewing and deliberating conflict of interest statements, and managing the documentation updates (self-assessment and final assessment) to the project's assessment folder.
+The lead reviewer serves as the primary security assessment facilitator, often coordinating 
+responses between the project and reviewers, ensuring communications are performed in a timely 
+fashion as described in the [guide](./).  The lead reviewer also responsible for coordinating 
```

Comment:
If I recall, I've always done COI (or asked the chairs to).  I feel like pushing this to the ever-changing lead is a way to make this more uneven, which feels like the wrong thing to outsource especially since this is so little work...